client/src/components/CardIcon.js

```diff
@@ -0,0 +1,59 @@
+        <path
+          d="M31.5 12.9018C34.3612 9.54181 38.745 7.41556 43.3125 7.41556C51.3975 7.41556 57.75 13.7681 57.75 21.8531C57.75 31.7674 48.8396 39.8473 35.3395 52.0891L35.3062 52.1193L31.5 55.5843L27.6938 52.1456L27.5901 52.0514C14.1294 39.8192 5.25 31.7502 5.25 21.8531C5.25 13.7681 11.6025 7.41556 19.6875 7.41556C24.255 7.41556 28.6388 9.54181 31.5 12.9018ZM31.5 48.4968L31.7625 48.2343C44.2575 36.9206 52.5 29.4393 52.5 21.8531C52.5 16.6031 48.5625 12.6656 43.3125 12.6656C39.27 12.6656 35.3325 15.2643 33.9675 18.8606H29.0588C27.6675 15.2643 23.73 12.6656 19.6875 12.6656C14.4375 12.6656 10.5 16.6031 10.5 21.8531C10.5 29.4393 18.7425 36.9206 31.2375 48.2343L31.5 48.4968Z"
+        />
+      )
+    case 'calendar':
+      return(
+        <path
+          d="M47.25 10.5H49.875C52.7625 10.5 55.125 12.8625 55.125 15.75V52.5C55.125 55.3875 52.7625 57.75 49.875 57.75H13.125C10.2113 57.75 7.875 55.3875 7.875 52.5V15.75C7.875 12.8625 10.2113 10.5 13.125 10.5H15.75V5.25H21V10.5H42V5.25H47.25V10.5ZM13.125 26.25V52.5H49.875V26.25H13.125ZM49.875 21H13.125V15.75H49.875V21ZM31.5 31.5H18.375V44.625H31.5V31.5Z"
+        />
+      )
+    case 'dress':
+      return(
+        <path
+          d="M36.463 58.2115C36.463 58.2115 35.0454 59.5015 32.5254 59.824C30.2417 60.0659 28.3517 59.6627 26.9342 59.2596C26.5652 58.3151 26.1454 57.2464 25.6993 56.1105C23.4798 50.4593 20.6075 43.1464 20.0829 41.1995C20.1617 41.7639 22.288 57.0021 24.1779 62.5652C23.7842 63.049 23.1542 63.2102 22.2879 63.1296C20.9346 62.9092 19.6653 62.3567 18.5271 61.8612C16.7511 61.0881 15.294 60.4538 14.3342 61.4365C12.9954 62.8071 9.2942 62.4846 8.42795 61.9202C9.77048 55.8435 11.0496 47.3002 11.3552 45.2594C11.3901 45.0258 11.4124 44.8774 11.4204 44.8277C10.8692 46.9239 7.64045 55.1477 5.3567 60.8721C2.60045 61.114 -0.0770546 59.7434 0.00169539 59.0983C8.74295 41.9252 10.1604 32.492 9.84545 26.8482C9.62381 23.4445 10.6108 19.7215 11.325 17.0271C11.6259 15.8922 11.8783 14.9398 11.9717 14.2707C12.2079 12.0132 10.7904 8.78816 10.7904 8.78816C10.2331 5.98698 13.2291 5.15493 16.1708 4.33795C17.8012 3.88512 19.4151 3.43693 20.398 2.66063C20.7917 1.45126 21.5792 0 21.5792 0L22.2879 0.241876C21.5625 2.33077 21.9596 3.67133 22.3218 4.89444C22.5888 5.79573 22.8368 6.63324 22.603 7.6594C22.4749 8.31484 22.2386 8.86797 21.9916 9.44631C21.4788 10.6465 20.9197 11.9554 21.1855 14.5126C21.3164 15.7865 21.7878 17.0466 22.2995 18.4144C23.0184 20.336 23.8169 22.4704 23.8629 25.1551C23.8873 26.154 23.7757 27.1065 23.6592 28.1012C23.3995 30.318 23.1152 32.7446 24.2567 36.362C26.4617 43.3764 31.1867 53.2127 36.463 58.2115Z"
+        />
+      )
+      return <path />
+  }
+}
+
+const Icon = ({
+  name = '',
+  style = {},
+  fill = "#441F0E",
+  width = "63",
+  height="63",
+  className = "",
+}) => (
+  <svg
+    width={width}
+    style={style}
+    height={width}
+    viewBox={getViewBox(name)}
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    {getPath(name)}
+  </svg>
+)
+
+
+export default Icon;
```

Comment:
Can you delete all this? If you need an icon, then you're going to use the `Icon` component that Kelly is working on!